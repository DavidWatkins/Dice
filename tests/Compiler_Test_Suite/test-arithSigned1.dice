class test {
	public void main(char[][] args) {
		print(-15/3);
		print(15/-3);
		print(-15.0/3.0);
		print(15.0/-3.0);
	}
}