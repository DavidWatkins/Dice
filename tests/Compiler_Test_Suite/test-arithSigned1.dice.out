-5-5-5.000000-5.000000